mne/__init__.py

```diff
@@ -98,6 +98,7 @@
 from . import viz
 from . import decoding
 from . import realtime
+from . import testing
```

Comment:
Will do. It's a high priority for me so I hope to get it done it time for release.